navigator/frontend/src/ui-core/src/ParameterForm/index.tsx

```diff
@@ -728,7 +782,8 @@ class TypeConInput extends React.Component<TypeConInputProps, TypeConInputState>
     const { dataType } = this.state;
 
     if (dataType === undefined) {
-      return <em>Loading type {parameter.name.name}...</em>;
+      console.log('RoDer ' + parameter.name.name );
+      return <em>JCVD Loading type {parameter.name.name}...</em>;
```

Comment:
What is JCVD? [Jean-Claude Van Damme](https://www.google.com/search?q=jcvd)?

Reply:
:S

removed